src/Module/Login.php

```diff
@@ -347,4 +378,56 @@ public static function form($return_path = null, $register = false, $hiddens = [
 
 		return $o;
 	}
+
+	/**
+	 * Get the URL to the register page and add OpenID parameters to it
+	 */
+	private static function getRegisterURL()
+	{
+		if (empty(Session::get('openid_identity'))) {
+			return 'register';
+		}
+
+		$args = [];
+		$attr = Session::get('openid_attributes') ?? [];
+
+		if (is_array($attr) && count($attr)) {
```

Comment:
The default value set to empty array above means that `$attr` will always be an array because we control it, which makes this condition useless.

Reply:
Although we checked if the value isn't empty - but it could be not empty but not an array.

Reply:
Not really, it isn't a user-supplied value, we have complete control over this session variable. Unless we directly assign a user-provided value to this session variable, but we wouldn't do that, would we?

Reply:
I haven't checked if the return value of the OpenID library that we are using is always returning an array.